src/app/content/components/__snapshots__/PrevNextBar.spec.tsx.snap

```diff
@@ -511,8 +523,24 @@ exports[`PrevNextBar renders only \`prev\` element correctly 1`] = `
     </svg>
     Previous
   </a>
-  <span
-    aria-hidden={true}
-  />
+  <a
```

Comment:
this test is supposed to be testing when only the prev element is rendered, now we've added another page so it is no longer testing the right thing.

maybe we should move our new page to not be at the end of the book?